scripts/gee

```diff
@@ -5108,7 +5197,7 @@ function _gee_completion() {
     2)
       prev="\${COMP_WORDS[COMP_CWORD-1]}"
       case "\${prev}" in
-        diff|unpack|revert)
+        diff|log|unpack|vimdiff|revert)
```

Comment:
is `log` here correct?

Reply:
It should be!  But my testing exposed a bug in my implementation of `gee log` such that it wasn't handling filename arguments correctly.  `gee log` is now fixed.